ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/customtile_detail.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Custom Tile URL</title>
+</head>
+<body>
+	<h1>Custom Tile Server Configuration</h1>
+	<p>
+		The <i>Custom Tile Maps</i> provides functionality to add map that uses web tiling scheme.
+	</p>
+	<p>
+		Here is the explanation of how to add, configure and remove <i>Custom Tile</i> 
```

Comment:
I'd remove the sentence "Here is the explanation of how to add, configure and remove <i>Custom Tile</i>...". It doesn't add information.